help_to_heat/frontdoor/routing/__init__.py

```diff
@@ -51,5 +51,5 @@ def calculate_journey(answers, to_page, from_page=None):
 
 class CouldNotCalculateJourneyException(Exception):
     def __init__(self, from_page, to_page, partial_journey):
-        super().__init__(f"Could not calculate a journey from {from_page} to {to_page}")
+        super().__init__(f"Could not calculate a journey from {from_page} to {to_page}. Route tried: {partial_journey}")
```

Comment:
Does Python handle list -> string nicely by default?

Reply:
did double check this (which is why I didn't do this initially) and yes, it seems so
![image](https://github.com/user-attachments/assets/5a5c0153-7ceb-405e-9ccb-25a09d4e85bb)